build/Dockerfile

```diff
@@ -2,29 +2,33 @@
 ARG BUILD_OS=debian
 ARG NGINX_PLUS_VERSION=R31
 ARG DOWNLOAD_TAG=edge
+ARG DEBIAN_FRONTEND=noninteractive
 
 
 ############################################# Base images containing libs for Opentracing and FIPS #############################################
-FROM opentracing/nginx-opentracing:nginx-1.25.3 as opentracing-lib
-FROM opentracing/nginx-opentracing:nginx-1.25.3-alpine as alpine-opentracing-lib
-FROM ghcr.io/nginxinc/alpine-fips:0.1.1-alpine3.18 as alpine-fips
+FROM opentracing/nginx-opentracing:nginx-1.25.3@sha256:426c2ff410f637eb243363f58c118254b5663e90224e5ab3c99d985eea50ec18 as opentracing-lib
```

Comment:
can dependabot update these SHAs as well to keep us up to date?

Reply:
Should be able to the "Pinned-Dependencies" recommendation is to use this syntax
[Dockerfile](https://github.com/ossf/scorecard/blob/main/cron/internal/controller/Dockerfile) & [dependabot.yml](https://github.com/ossf/scorecard/blob/55b6b7686d64d0f87b96c2f53742df1ae109b570/.github/dependabot.yml#L65)